src/components/engineService.ts

```diff
@@ -139,6 +153,47 @@ export class EngineService {
 		this.#noraRustEngineExecutableUri = message.noraRustEngineExecutableUri;
 	}
 
+	public async getCodemodList(): Promise<Readonly<CodemodEntry[]>> {
+		const executableUri = this.#noraNodeEngineExecutableUri;
+
+		if (!executableUri) {
+			throw new EngineNotFoundError('Node engine not found');
+		}
+
+		const childProcess = spawn(
```

Comment:
`spawn` will be closed automatically when the process finishes right?

Reply:
Yes, it should